### PR TITLE
Fix theming issue with drawer + dropdown menu

### PR DESCRIPTION
**Pull request by WesleyKapow**
* Added manual headlessui-portal-root to get drawer added within CTWProvider which defines theme CSS vars.
* Added a CTWProviderRef to the useCTW context and explicitly told dropdown-menu to use that for portal placement.
* Added custom theme example to App.tsx
* Fixed a few minor theming issues with colors in a few spots.

Screen Shot 2022-10-07 at 3 24 25 PM
Screen Shot 2022-10-07 at 3 24 43 PM

Screen Shot 2022-10-07 at 3 25 19 PM

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,12 +2,31 @@ import "./App.css";
...
+      black: "#1A2848",
+      light: "#4A4A4A",
+      lighter: "#999999",
+    },
+    bg: {
+      light: "#F1F1F1",
+    },
+  },
+};
+
```
Comment (luthfib) [inline src/App.tsx:26]: This is good, that way we have a visible example in our library of how it's done

File: src/App.tsx [hunk 2]
```diff
@@ -2,12 +2,31 @@ import "./App.css";
...
 function App() {
   return (
-    <CTWProvider env="dev" authToken={VITE_AUTH_TOKEN} theme={{}}>
+    <CTWProvider env="dev" authToken={VITE_AUTH_TOKEN} theme={theme}>
```
Comment (luthfib) [inline src/App.tsx:29]: I wonder if at some point we want env to be dictated by the .env file.

**Review by luthfib — APPROVED** [review]
LGTM, left a minor comment that we may not even want to address, just a thought

File: src/App.tsx (hunk 2 above)
Comment (WesleyKapow) [reply src/App.tsx:29]: Ya not a bad idea. The whole auth and env stuff will likely change when we setup storybook. Gonna leave as is for now.

**Pull request merged**